site/_posts/2024-02-29-localized-variables.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: Localized variables
+lang: en
+---
+
+Polyglot allows you to have different pages for different languages in your Jekyll site. For example, one could have a page `about.md` in English and another `about.md` in Spanish with completely different layouts. But if you want to have the same layout for both pages, you can use localized variables. This is a way to have different data for different languages in your Jekyll site, but using the same layout for all languages.
+
+As an example I will use a [template site](https://github.com/george-gca/multi-language-al-folio) created with Polyglot.
```

Comment:
👍  love the cross-promotion and example!

Reply:
Thought it would be useful to use as an example an actual site built with Polyglot, and since I used all the stuff that I wrote about and know why/where, it was easier to do with it.